src/clients/api/queries/getAuthentication/index.ts

```diff
@@ -0,0 +1,17 @@
+import { restService } from 'utilities';
+
+const getAuthentication = async (accountAddress: string | undefined): Promise<boolean> => {
```

Comment:
To follow how other query functions define their arguments, I'd suggest using an object. I also think `accountAddress` should never be `undefined`, as otherwise the logic says that an `undefined` address is not authorized which is technically incorrect. Instead, we can just disable the hook when the address passed is `undefined` (more on that later).
```suggestion
const getAuthentication = async ({ accountAddress }: { accountAddress: string }): Promise<boolean> => {
```